packages/webapp/src/pages/play.tsx

```diff
@@ -202,15 +248,18 @@ const Play: FablePage = ({ isHydrated }) => {
             </>
           )}
 
-          <div className="relative row-span-6 rounded-xl rounded-t-none border border-t-0 bg-base-300 shadow-inner">
-            <p className="z-0 m-2 font-mono font-bold"> ⚔️ p1 address </p>
-            <p className="-0 m-2 font-mono font-bold"> ♥️ 100 </p>
-            {/* <div className="absolute bottom-0 right-1/2 -mb-2 flex h-32 w-32 translate-x-1/2 items-center justify-center rounded-full border bg-slate-900  font-mono text-lg font-bold text-white">
-              100
-            </div> */}
-          </div>
+          <PlayerBoard playerAddress={playerAddress} playedCards={playerBattlefield} />
+
+          {createPortal(
+            <DragOverlay dropAnimation={dropAnimation}>
+                <BaseCard id={activeId as number} placement={CardPlacement.DRAGGED} />
+            </DragOverlay>,
+            document.body
+          )}
```

Comment:
Woops, forgot to check this, but basically, does this always need to exist, or can we just create it?

What's the mechanism through which this is hidden if there is no card being dragged?

Reply:
The necessary docs for this are here: https://docs.dndkit.com/api-documentation/draggable/drag-overlay#usage

the reason for it to exist it is as mentioned: 

> The <DragOverlay> component should remain mounted at all times so that it can perform the drop animation. If you conditionally render the <DragOverlay> component, drop animations will not work.

Reply:
So the `DragOverlay` itself will hide the component, right?